libs/s25main/desktops/dskHostGame.cpp

```diff
@@ -759,6 +764,36 @@ void dskHostGame::Msg_MsgBoxResult(const unsigned msgbox_id, const MsgboxResult
                 UpdateGGS();
         }
         break;
+        case 10: // Economy Mode - change Addon Setttings
+        {
+            if(mbr == MSR_YES)
+            {
+                gameLobby->getSettings().setSelection(AddonId::PEACEFULMODE, true);
```

Comment:
probably we should not imply peaceful mode here - still could be interesting to fight while trying to achieve the economy goal?

Reply:
Upon further consideration, I agree. I had originally added this, because at first I didn't see how you could keep addon settings between games. I will revert that. 

I will for now keep the other check that you don't play with peaceful mode in attack based games, unless you think that's superfluous.

Maybe in the future it would be nice to be able to save multiple addon profiles. Would be convenient if there are different game modes you like to play with

Reply:
@Flow86 It isn't implied. There is a message box asking you whether to enable peaceful mode. I think that is a good idea as that might be what is often wanted. If the message box says exactly what it changes I'd keep it.

Reply:
I don't know who has seniority here or something. I put the message box back in and made the text much clearer. Just decide amongst yourselves and possibly revert the commit if you decide against it.

Reply:
ah ok then it should be fine if you are asked for it.